sse2neon.h

```diff
@@ -4706,7 +4783,12 @@ FORCE_INLINE __m128i _mm_packus_epi16(const __m128i a, const __m128i b)
 // https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm_pause
 FORCE_INLINE void _mm_pause()
 {
+#if defined(_MSC_VER)
+    __dmb(_ARM64_BARRIER_ISHST);
```

Comment:
This dmb + yield combo is based on the definition of YieldProcessor() in winnt.h. I can make no guarantees as to the wisdom this implementation choice. Plainly it does not match the inline assembly below (but I suspect that's not ideal either...).